### ISSUE TITLE
Watching replays on different speeds affects superweapon timers

### ISSUE THREAD

**Issue by r34ch** [OpenRA/OpenRA#10466]
On release, RA mod.

If you toggle a replay between the different speeds the timer in the top corner is affected.

![timer](https://cloud.githubusercontent.com/assets/3636652/12218411/6c434678-b716-11e5-9bef-cc2c039fe62d.jpg)
Here it says 0:19 seconds remaining when the actual time is closer to 7 minutes when viewing a replay on max speed.

**Comment by r34ch** [comment]
```
21:39 <+pchote> that timer thing is a one line fix
21:40 <+pchote> in the FormatTime calls pass it the timestep encoded in the lobby info
```

**Comment by pchote** [comment]
I'm not sure how you get the timer in the top left (it has never shown for me?!), but I fixed it for the stats menu in #10478.
